EPlast/EPlast/wwwroot/css/organization.css

```diff
@@ -0,0 +1,388 @@
+﻿* {
+    margin: 0;
+    padding: 0;
+}
+
+.profile-icon {
+    background-image: url('http://plast-dev.sitegist.net/images2/eplast/menu-profile.png');
+}
+
+.decision-icon {
+    background-image: url('http://plast-dev.sitegist.net/images2/eplast/plast-document-36.png');
+}
+
+.dictionary-icon {
+    background-image: url('http://plast-dev.sitegist.net/images2/eplast/menu-handbook.png');
+}
+
+.admin-icon {
+    background-image: url('http://plast-dev.sitegist.net/images2/eplast/menu-admin.png');
+}
+
+.cabinet-icon {
+    background-image: url('http://plast-dev.sitegist.net/images2/eplast/menu-dashboard.png');
+}
+
+.sidebar-logo {
+    padding: 0px;
+    width: 40px;
+    height: 40px;
+    margin-top: 10px;
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center center;
+    opacity: 0.7;
+    cursor: pointer;
+}
+
+.plast-logo {
+    width: 40px !important;
```

Comment:
using !important is considered to be bad practice and it is usually avoided if possible, try to use more precise selector instead